x

```diff
@@ -0,0 +1,1549 @@
+Using deployment integration
```

Comment:
Curious: what is this file (looks like a log to me) and are there any credentials exposed in this file?

Reply:
My bad.  I removed that file.
No credentials.  That was just the output of a `dcpdig @upload` run.